app/emails/users.py

```diff
@@ -89,11 +89,11 @@ def generate_user_creation_token(name, email_address, user_type, **unused):
     return token
 
 
-def send_account_activation_email(name, email_address, user_type):
+def send_account_activation_email(name, email_address, user_type, url):
     token = generate_user_creation_token(name=name, email_address=email_address, user_type=user_type)
     url = '{}{}/create-user/{}'.format(
```

Comment:
I don't think aslant would let you get away with this - having a var the same name as a parameter :)